x-pack/docs/en/ilm/update-lifecycle-policy.asciidoc

```diff
@@ -1,5 +1,186 @@
 [[update-lifecycle-policy]]
 == Update lifecycle policy
 
-Discuss how one updates an existing policy (when allowed, when not),
- as well as all the stuff about switching policies on indices.
+Updating existing ILM policies is useful to fix mistakes or change
+strategies for newly created indices. Existing indices that are being
+managed by a policy that changes will be modified as well. So that
+phases currently being executed are not interrupted at dangerous times, updates
+to policies will not affect the current phase that is being executed.
+
+There are three scenarios for examining the behavior updating policies and
+their effects on policy execution on indices.
+
+A. Effects of a policy update on an index that is not being managed by
+   the modified policy
+ - Nothing occurs on the index since it is unmanaged. If an index is assigned to
+    the policy, it will be assigned the latest version of that policy
+B. Effects of updating a specific phase's definition with an index currently executing the policy.
+ - Indices preserve the phase definition from the latest policy version that existed
+     at the time that it entered that phase. So if an index is in the `hot` phase, and the
+     `hot` phase's definition is updated in the meantime, then the index will not experience
+     any changes until it completes the previous version's hot phase and continues to the next
+     defined phase (as defined by the latest version).
+C. Effects of switching which policy is managing an index
+ - This behaves much like a policy update, but instead of just switching to a different
+     version, it switches to a different policy (the latest version of that policy).
+
+
+=== A. Update Policy With No Indices
```

Comment:
I wonder if we should combine these sections with the descriptions above so a user can see the description and example of how to do it together?

Reply:
I agree. I've moved the expanded scenarios within each section